sdk/sphere/Azure.ResourceManager.Sphere/api/Azure.ResourceManager.Sphere.netstandard2.0.cs

```diff
@@ -0,0 +1,611 @@
+        public static Azure.ResourceManager.Sphere.Models.DeviceInsight DeviceInsight(string deviceId = null, string description = null, System.DateTimeOffset startTimestampUtc = default(System.DateTimeOffset), System.DateTimeOffset endTimestampUtc = default(System.DateTimeOffset), string eventCategory = null, string eventClass = null, string eventType = null, int eventCount = 0) { throw null; }
+        public static Azure.ResourceManager.Sphere.ImageData ImageData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, string image = null, string imageId = null, string imageName = null, Azure.ResourceManager.Sphere.Models.RegionalDataBoundary? regionalDataBoundary = default(Azure.ResourceManager.Sphere.Models.RegionalDataBoundary?), System.Uri uri = null, string description = null, string componentId = null, Azure.ResourceManager.Sphere.Models.ImageType? imageType = default(Azure.ResourceManager.Sphere.Models.ImageType?), Azure.ResourceManager.Sphere.Models.ProvisioningState? provisioningState = default(Azure.ResourceManager.Sphere.Models.ProvisioningState?)) { throw null; }
+        public static Azure.ResourceManager.Sphere.ProductData ProductData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, string description = null, Azure.ResourceManager.Sphere.Models.ProvisioningState? provisioningState = default(Azure.ResourceManager.Sphere.Models.ProvisioningState?)) { throw null; }
+        public static Azure.ResourceManager.Sphere.Models.ProofOfPossessionNonceResponse ProofOfPossessionNonceResponse(string certificate = null, Azure.ResourceManager.Sphere.Models.CertificateStatus? status = default(Azure.ResourceManager.Sphere.Models.CertificateStatus?), string subject = null, string thumbprint = null, System.DateTimeOffset? expiryUtc = default(System.DateTimeOffset?), System.DateTimeOffset? notBeforeUtc = default(System.DateTimeOffset?), Azure.ResourceManager.Sphere.Models.ProvisioningState? provisioningState = default(Azure.ResourceManager.Sphere.Models.ProvisioningState?)) { throw null; }
+        public static Azure.ResourceManager.Sphere.Models.SignedCapabilityImageResponse SignedCapabilityImageResponse(string image = null) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CapabilityType : System.IEquatable<Azure.ResourceManager.Sphere.Models.CapabilityType>
```

Comment:
This name is quite generic, could we be more specific on this type name?

Reply:
fixed in latest commit